lib/tasks/update-pension-calc-tool-name.rake

```diff
@@ -0,0 +1,13 @@
+namespace :corporate_partners do
+  desc 'Update tool name from "Pensions calculator" to "Pension calculator"'
+  task update_tool_name: :environment do
+    records_to_update = CorporatePartner.where(tool_name: 'Pensions calculator')
+    puts "record ids: #{records_to_update.map(&:id)}"
+
+    CorporatePartner.transaction do
+      records_to_update.update_all(tool_name: 'Pension calculator')
```

Comment:
Note: `update_all` skips all `ActiveRecord` validations and does not update the `updated_at` field, which can be very handy in situations like this actually. On the top of that, it produces an atomic SQL statement, that means it does not need to be wrapped in a transaction block. It will also return the number of records updated.